app.js

```diff
@@ -40,6 +40,7 @@ app.use('/our-program', indexRouter);
 app.use('/apply', indexRouter);
 app.use('/legal-disclosure', indexRouter);
 app.use('/privacy-policy', indexRouter);
+app.use('/tracks/:trackId(pm|se|ixd|ai)', indexRouter);
```

Comment:
I'd use the complete names for the tracks for SEO and readability reasons. 

```suggestion
app.use('/tracks/:trackId(product-management|software-engineering|interaction-design|artificial-intelligence)', indexRouter);
```

Reply:
👍  (need to change on react side as well)